integration_tests/pages/refer/checkAnswers.ts

```diff
@@ -40,9 +40,22 @@ export default class CheckAnswersPage extends Page {
     this.username = username
   }
 
-  confirmDetailsAndSubmitReferral(referral: Referral) {
+  confirmDetailsAndSubmitReferral() {
+    // Need to stub the call to get the referral twice because
+    // the first stub is for the submit method and
+    // the second stub is for the complete page
+    cy.task('stubReferralWithScenario', {
+      newScenarioState: 'submitted',
+      referral: this.referral,
+      scenarioName: 'submit',
```

Comment:
We don't need a required status of `"Started"` here? How does it know to use the scenario below if this one doesn't have an exclusive status requirement?

Reply:
Make `requiredScenarioState` required (default `"Started"`)